mkdocs.yml

```diff
@@ -2,11 +2,12 @@ docs_dir: docs/book
 site_dir: docs/html
 pages:
     - index.md
-    - Intro: v2/intro.md
+    - Introduction: v2/intro.md
```

Comment:
This should be used everywhere! 👍

Reply:
Yeah - I noticed that the intro page is not linked in the current TOC, which is unfortunate, as that has the "getting started" guide!

You can still get to it via the `Next` link in the home page, but not great!